### PR TITLE
Passing vcf indices correctly in M2 wdls

### PR DESCRIPTION
**Pull request by davidbenjamin**
This is the bug fix for @eitanbanks's pon.  @LeeTL1220 can you review if tests pass?  If they fail I will fix ASAP.

### CODE REVIEW

**Comment by LeeTL1220** [comment]
@davidbenjamin I think this affects more than the PoN.

**Comment by LeeTL1220** [comment]
Are you sure that @eitanbanks was specifying `compress` of `true`?  If it was `false`, then I believe we've misdiagnosed the issue.  Doubly odd, since `false` is the default.

File: scripts/mutect2_wdl/unsupported/mutect2_opt.wdl [hunk 1]
```diff
@@ -182,14 +182,15 @@ workflow Mutect2 {
...
+                compress = compress,
                 gatk_override = gatk_override,
                 gatk_docker = gatk_docker,
                 preemptible_attempts = preemptible_attempts,
                 disk_space = m2_per_scatter_size,
                 auth = auth
         }
 
-        Float sub_vcf_size = size(M2.output_vcf, "GB")
+        Float sub_vcf_size = size(M2.vcf, "GB")
```
Comment (LeeTL1220) [inline scripts/mutect2_wdl/unsupported/mutect2_opt.wdl]: I prefer `output_vcf` or something more descriptive than `vcf`... `raw_vcf`?

**Comment by davidbenjamin** [comment]
> Are you sure that @eitanbanks was specifying compress of true? If it was false, then I believe we've misdiagnosed the issue. Doubly odd, since false is the default.

@LeeTL1220 False is and was the default for all final outputs, but the intermediate scatter vcfs were hard-coded to be compressed.

**Pull request merged**